lib/cubits/register/register_cubit.dart

```diff
@@ -21,6 +23,8 @@ class RegisterCubit extends Cubit<RegisterState> {
       encodePasscode(state.passcode!),
     );
 
+    sl<FirebaseAnalyticsEventLogging>().signUpEvent();
```

Comment:
Should be below the if clause (to prevent firing a sign up event in the case of an unsuccessful registration).